src/Serializer/AbstractItemNormalizer.php

```diff
@@ -417,7 +489,7 @@ protected function getAttributeValue($object, $attribute, $format = null, array
      *
      * @return string|array
      */
-    private function normalizeRelation(PropertyMetadata $propertyMetadata, $relatedObject, string $resourceClass, string $format = null, array $context)
+    protected function normalizeRelation(PropertyMetadata $propertyMetadata, $relatedObject, string $resourceClass, string $format = null, array $context)
```

Comment:
Not sure that this should be protected. Other changes in this file are not needed